static/js/util/util.js

```diff
@@ -249,6 +249,6 @@ export function calculateDegreeInclusions(profile: Profile) {
 /**
  * Calls an array of functions in series with a given argument and returns an array of the results
  */
-export function callFunctionArray(functionArray, arg) {
+export function callFunctionArray(functionArray: Function[], arg: any): any[] {
```

Comment:
Is it possible to specify the type of `functionArray`? I'm thinking about something with type `A => R` with arg: `A` and the return value `R[]`